modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -159,6 +159,15 @@ These adjustments are only applied to cleartext, edge-terminated, and re-encrypt
 
 For request headers, these adjustments are applied only for routes that have the `haproxy.router.openshift.io/h1-adjust-case=true` annotation. For response headers, these adjustments are applied to all HTTP responses. If this field is empty, no request headers are adjusted.
 
+|`httpCompression`
+|`httpCompression` defines the policy for HTTP traffic compression. The optional `ROUTER_ENABLE_COMPRESSION` field turns on compression for the MIME types, which are specified in the `mimeTypes` variable. Compression is not enabled by default.
+
+You can configure compression of MIME types using the environment variables:
+
+* `ROUTER_ENABLE_COMPRESSION` if set to `true` or `TRUE`, then the body of the response is compressed when possible.
```

Comment:
I think this should flow as a sentence or a colon should be put after the variable to follow similar examples in the same module. For example:

`ROUTER_ENABLE_COMPRESSION`: If set to true. . .

Alternatively:

`ROUTER_ENABLE_COMPRESSION`compresses the body of the response when set to `true` or `TRUE` when possible.